src/WinRMSharp/Protocol.cs

```diff
@@ -72,6 +72,33 @@ internal Protocol(ITransport transport, IGuidProvider guidProvider, ProtocolOpti
             MaxEnvelopeSize = options?.MaxEnvelopeSize ?? DefaultMaxEnvelopeSize;
         }
 
+        /// <inheritdoc cref="IProtocol.Identify" />
+        public async Task<IdentifyResponse> Identify()
+        {
+            Envelope envelope = new Envelope
+            {
+                Header = new Header(),
+                Body = new Body
+                {
+                    Identify = new Identify()
+                }
+            };
+
+            XDocument root = await Send(envelope);
+
+            XNamespace ns = XNamespace.Get("http://schemas.dmtf.org/wbem/wsman/identity/1/wsmanidentity.xsd");
+
+            return new IdentifyResponse
+            {
+                ProtocolVersion = root.Descendants(ns + "ProtocolVersion").Single()?.Value

```

Comment:
Shouldn't this be `SingleOrDefault` or `FirstOrDefault`? Won't `Single` throw an exception if no element is found?